pkgs/development/libraries/libvirt/default.nix

```diff
@@ -8,6 +8,7 @@
 , enableXen ? false, xen ? null
 , enableIscsi ? false, openiscsi
 , enableCeph ? false, ceph
+, enableHyperv ? true, openwsman
```

Comment:
It seems wrong that HyperV is enabled by default when the rest of the drivers are not. Personally, I'd rather have the default be they are all enabled and then if someone needs a smaller size, they would disable them. However, I don't know much about this package.

cc @volth @fpletz

Reply:
Please disable this by default because this will be used by a very tiny amount of people.